sprints/dashboard/utils.py

```diff
@@ -240,16 +240,16 @@ def get_sprints(conn: CustomJira, board_id: int) -> List[Sprint]:
     return conn.sprints(board_id, state='active, future')
 
 
-def get_sprint_number(previous_sprint: Sprint) -> int:
+def get_sprint_number(sprint: Sprint) -> int:
```

Comment:
This is not related, but the wording of this function was confusing - its purpose has changed after a refactor, but we forgot to rename internal variables.